elements/rh-spinner/test/rh-spinner.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { expect, html } from '@open-wc/testing';
+import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+import { RhSpinner } from '@rhds/elements/rh-spinner/rh-spinner.js';
+
+const element = html`
+  <rh-spinner></rh-spinner>
+`;
+
+describe('<rh-spinner>', function() {
+  it('should upgrade', async function() {
```

Comment:
ideally, we should do all the [same tests as pfe-spinner](https://github.com/patternfly/patternfly-elements/blob/main/elements/pfe-spinner/test/pfe-spinner.spec.ts), particularly the computed width tests